samples/client/petstore/scala-akka/src/test/scala/PetApiTest.scala

```diff
@@ -118,6 +119,5 @@ class PetApiTest extends AsyncFlatSpec with Matchers {
         }
       }
   }
-  */
+   */
```

Comment:
Can you remove this commented out block as well please?

Reply:
That is a good catch, thanks. 
On the other hand I was not able to reproduce the CircleCI bug and I would really like some help on that, I'm not sure to understand what the `circe_parallel` script is supposed to do, but I can't run it locally.